saw-core-coq/coq/Makefile

```diff
@@ -1,13 +1,23 @@
-all: build
+all: Makefile.coq
 
 Makefile.coq: _CoqProject
 	coq_makefile -f $< -o $@
 
-build: Makefile.coq
-	make -f Makefile.coq
+include Makefile.coq
 
-clean: Makefile.coq
-	make -f Makefile.coq clean
+# If running in CI, always set $SAW=`which saw`. Otherwise, if saw is not in the
+# current path, fall back on cabal run saw
```

Comment:
Why is CI not allowed to use `cabal run saw`?

Also you might want to reword to something like "Always try to find saw in the path, otherwise fall back on cabal run saw [unless running in CI because ...]"